### ISSUE TITLE
Serde support

### ISSUE THREAD

**Issue by Boscop** [softdevteam/vob#9]
Please add optional support for serde's Serialize/Deserialize traits.
Like this:
```toml
serde = { version = "1.0", features = ["derive"], optional = true }
```
```rust
#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
struct Vob {
```

**Comment by ltratt** [comment]
Good idea -- I'll look into this.

**Comment by ltratt** [comment]
Now released as 1.1.0.
